### DISCUSSION TITLE
Export View file from vendor to resources

### DISCUSSION THREAD

**Discussion by pcodiant** [opcodesio/log-viewer#164] [Q&A] [answered]
Export view from **"vendor"** to the folder **"Resources/views"** for customization

Is there any way to get all the files and folder inside the root folder?

**Comment by arukompas** [comment, marked as answer]
Hey @pcodiant - at the moment the view resources are not meant to be exportable or configurable. The design of the Log Viewer can still change a lot in the near future and I wanted package users to be able to upgrade easily without any conflicts.

So, for now, customisation of the views is not possible :)
